Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1834,6 +1834,56 @@ def fetch_incidents_command_rec(start_time, last_time, page=1):
     return incidents, last_time
 
 
+def get_path_data():
+    """
+    Get path data from given relative file path, repository and organization corresponding to branch name if given.
+    Returns:
+        Outputs to XSOAR.
+    """
+    args = demisto.args()
+
+    relative_path: str = args.get('relative_path', '')
+    repo: str = args.get('repository') or REPOSITORY
+    organization: str = args.get('organization') or USER
+    branch_name: Optional[str] = args.get('branch_name')
+
+    url_suffix = f'/repos/{organization}/{repo}/contents/{relative_path}'
+    url_suffix = f'{url_suffix}?ref={branch_name}' if branch_name else url_suffix
+
+    headers = {
+        'Authorization': "Bearer " + TOKEN,
+        'Accept': 'application/vnd.github.VERSION.object',
```

Comment:
Why do we add this? is this needed? 
```suggestion
        'Accept': 'application/vnd.github.VERSION.object',
```

Reply:
For consistency, object returns the expected. I am afraid in future they will change their API to do something else if Accept was not given.